src/js/pinny.js

```diff
@@ -444,27 +453,23 @@
         },
 
         /**
-         * In iOS7 or below, when elements are focused inside pinny
+         * In iOS7/Android 4 or below, when elements are focused inside pinny
          * the keyboard obscures the input. We need to scroll back to
          * the element to keep it in view.
          */
         _handleKeyboardShown: function() {
-            if (iOS7orBelow) {
-                var activate = function() {
-                    this._showSpacer();
-                    this._scrollToTarget();
-                };
-
+            if (needsSpacer) {
                 this.$pinny.find(FOCUSABLE_INPUT_ELEMENTS)
-                    .on(events.focus, activate.bind(this))
+                    .on(events.focus, function() {
+                        this._showSpacer();
+                        this._scrollToTarget();
```

Comment:
This is pretty busted on Android 4.0 smartphones right now (and maybe on iOS smartphones too?)